src/code42cli/cmds/users.py

```diff
@@ -141,6 +148,17 @@ def update_user(
     "archive_size_quota",
 ]
 
+_bulk_user_move_headers = ["username", "org_id"]
+
+
+@users.command(name="move")
+@username_option("The username of the user to be moved.", required=True)
```

Comment:
Don't need the `be`, could just say:

`The username of the user to move`

Reply:
Updated.